packages/api-plugin-fulfillment-method-shipping-flat-rate/src/queries/getFlatRateFulfillmentRestrictions.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name getFlatRateFulfillmentRestrictions
+ * @method
+ * @memberof Fulfillment/NoMeteorQueries
```

Comment:
We should def remove this "NoMeteorQueries"

Reply:
Removed